source/TestFramework/NullableAttributes.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// ReSharper disable once CheckNamespace
+namespace System.Diagnostics.CodeAnalysis
```

Comment:
Wondering if this file should belong to Test lib?

Reply:
Yes, these are being copied over from nanoFramework.System.Runtime to resolve the strange running dependency that is breaking tests.